src/backend/database/migrations/tenant/2025_11_20_235951_add_sms_gateway_to_main_settings.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up() {
+        Schema::connection('tenant')->table('main_settings', function (Blueprint $table) {
+            $table->unsignedInteger('sms_gateway_id')->nullable()->comment('Selected SMS gateway ID: 2=AndroidGateway, 10=ViberGateway, 18=AfricasTalkingGateway, 26=TextbeeSmsGateway');
```

Comment:
I don't think we'll mange to keep this comment up to date 😉 It's to easy to forget to update it, if a new SMS Provider is added. So, unfortunately, I'd say let's remove it.